### ISSUE TITLE
GDBT example don't work

### ISSUE THREAD

**Issue by set92** [microsoft/nni#1718]
**Short summary about the issue/question**:
I cloned the whole repo and tried to run the example with 'nnictl create --config ./config.yml' like it says in the tutorial. I didn't change anything, but when I run it, the status of every trial was failed, so not sure if there is something missing in the tutorial, there is something that I'm not doing, or not sure where is the error.

**Brief what process you are following**: 
git clone https://github.com/microsoft/nni.git
cd examples/trials/auto-gbdt/
nnictl create --config ./config.yml
nnictl top

And every trial shows a FAILED status.

**nni Environment**:
- nni version: 1.1
- nni mode(local|pai|remote): local
- OS: manjaro
- python version: 3.6
- is conda or virtualenv used?: no
- is running in docker?: no

**Comment by liuzhe-lz** [comment]
Please check `trial.log` and `stderr` in `~/nni/experiments/<EXPERIMENT_ID>/trials/<TRIAL_ID>/`.
They should show why did trials fail.

**Comment by xuehui1991** [comment]
Could you post you trial.log and dispatcher.log?

By the way, have you install `lightgbm` and `pandas`?

**Comment by set92** [comment]
Fixed, thanks, it helped me the output of stderr that said that I didn't have installed lightgbm, and the thing was that I was using another python that I had installed. I created a symbolic link from python3.6 to python3 and done.

In the interface (http://10.12.11.13:8080/detail), the section of log only shows the log path, is normal? It should be cool if it could show it there the contents of the file too.
